scripts/popup.js

```diff
@@ -26,4 +26,96 @@ function view_all_function(){
 
 document.getElementById('save_now').onclick = save_now_function;
 document.getElementById('recent_capture').onclick = recent_capture_function;
-document.getElementById('first_capture').onclick = first_capture_function;
\ No newline at end of file
+document.getElementById('first_capture').onclick = first_capture_function;
+
+
+
+window.onload=function(){
+
+    chrome.runtime.sendMessage({message: "sendurl"}, function(response) {
+        
+        display_alexa_info(response.sent_url);
+        display_whois_info(response.sent_url);
+    });
+    
+};
+
+function display_alexa_info(url){
+    var alexa_url = 'http://data.alexa.com/data?cli=10&dat=snbamz&url=';
+		
```

Comment:
@rgpt  thanks for the review. I agree with you on that. The same is in case of the Whois information. However for the purpose of making a working example I felt free apis suffice.